.github/workflows/CI.yml

```diff
@@ -47,8 +47,8 @@ env:
   TESTING_FRAMEWORK_REPO: aws-observability/aws-otel-test-framework
   GITHB_RUN_ID: ${{ github.run_id }}
   DDB_TABLE_NAME: BatchTestCache
-  MAX_JOBS: 5
-  BATCH_INCLUDED_SERVICES: EKS,EKS_ARM64
+  MAX_JOBS: 100
+  BATCH_INCLUDED_SERVICES: EC2
```

Comment:
was ECS in this list before?

Reply:
Yeah it was. I just didn't see the need to add it. It can be changed if someone wants to run ECS tests on dev branch. I am going to cancel this CI run anyways immediately after merge.